src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.nav__list {
+  padding: 23px 0;
+  margin: 0;
+  display: flex;
+  list-style: none;
+  white-space: nowrap;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active,
+.laptop,
+.nav__item:hover {
```

Comment:
```suggestion
.is-active,
.nav__item:hover {
```
this shouldn't be highlighted by default. move hover from `li` to `a` instead
<img width="667" alt="image" src="https://user-images.githubusercontent.com/53611910/206932331-622dc84c-c50b-499e-beb8-7bbd14998b1c.png">